src/frontends/pytorch/src/op/scaled_dot_product_attention.cpp

```diff
@@ -15,13 +15,15 @@
 #include "openvino/op/matmul.hpp"
 #include "openvino/op/multiply.hpp"
 #include "openvino/op/range.hpp"
+#include "openvino/op/reshape.hpp"
 #include "openvino/op/select.hpp"
 #include "openvino/op/shape_of.hpp"
 #include "openvino/op/softmax.hpp"
 #include "openvino/op/sqrt.hpp"
 #include "openvino/op/squeeze.hpp"
 #include "openvino/op/transpose.hpp"
 #include "openvino/op/unsqueeze.hpp"
+#include "openvino/op/util/framework_node.hpp"
```

Comment:
why this include is needed?

Reply:
But it should be already included in `utils.hpp`

Reply:
I just pushed an update to include this in utils.hpp